cap/views.py

```diff
@@ -49,12 +49,12 @@ def ping(service=None):
         elif service == 'files':
             default_location = Location.get_default().uri
             test_file_path = join(default_location, 'test.txt')
-            with open(test_file_path, 'r'):
+            with open(test_file_path, 'w'):
                 pass
 
-        return {'message': 'OK'}, 200
+        return f'{service} OK!', 200
```

Comment:
Either one will work:

(1) 'OK'/'ERROR' + statuses
(2) we leave `{message: 'OK'/'ERROR' }` as dict, but pass it to `jsonify` so there are no errors